examples/next-prisma-starter-websockets/src/pages/_app.tsx

```diff
@@ -5,7 +5,7 @@ import { trpc } from 'utils/trpc';
 
 const MyApp: AppType = ({ Component, pageProps }) => {
   return (
-    <SessionProvider session={pageProps.session}>
+    <SessionProvider session={(pageProps as any).session}>
```

Comment:
We should be able to fix this without `any`

Reply:
@KATT Only if we create a type for it. But this is not intended behavior. `next@12.3.1` [fixes this](https://github.com/vercel/next.js/pull/40391). Do I just bump the version?

Reply:
Yes we likely need to bump the version on `next` and `next-auth`

Reply:
It should be 

const MyApp: AppType<{ session: Session | null }> = () => ...

Reply:
I will try bumping the version and if it doesn't work already I will add the type. Is that ok?

Reply:
What Julius says above is probably right -- I didn't look into _why_ it started failing, I just noted the issue down